libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -210,6 +212,12 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
     private _outsideClickSubscription = Subscription.EMPTY;
     private _dataSourceSubscription = Subscription.EMPTY;
 
+    /** @hidden */
+    private _isFocused = false;
+
+    /** An RxJS Subject that will kill the stream upon component’s destruction (for unsubscribing)  */
+    private readonly _onDestroy$: Subject<void> = new Subject<void>();
```

Comment:
call to onDestroy$.next() and complete() methods in `ngOnDestroy` needed, right?

Reply:
fixed now.